src/main/java/camp/nextstep/edu/step3/Hit.java

```diff
@@ -0,0 +1,46 @@
+package camp.nextstep.edu.step3;
+
+import java.util.Arrays;
+
+public enum Hit {
+    ZERO(0, 0),
+    ONE(1, 0),
+    TWO(2, 0),
```

Comment:
```suggestion
    MISS(3, 0),
```
구지 세 상태를 분리할 필요가 있을까요?

Reply:
제가 해당 enum 을 만들때 인터넷 로또를 생각하면서 만들었는데요
동X복권 사이트 에서 사용자가 로또 확인하면 안맞거나 , 한개 , 두개 맞아도 보여주자나요 
그런 의미에서 만들었습니다. 그래서 세 상태를 놔뒀습니다~